### PR TITLE
More redirect changes 

### PR DESCRIPTION
**Pull request by hdhalter**
### Description
More redirect updates.

### Issues Resolved
_List any issues this PR will resolve, e.g. Closes [...]._

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _security/access-control/index.md [hunk 1]
```diff
@@ -21,9 +21,9 @@ Term | Description
...
-Role mapping | Users assume roles after they successfully authenticate. Role mappings, well, map roles to users (or backend roles). For example, a mapping of `kibana_user` (role) to `jdoe` (user) means that John Doe gains all the permissions of `kibana_user` after authenticating. Likewise, a mapping of `all_access` (role) to `admin` (backend role) means that any user with the backend role of `admin` gains all the permissions of `all_access` after authenticating. You can map each role to many users and/or backend roles.
+Role mapping | Users assume roles after they successfully authenticate. Role mappings map roles to users (or backend roles). For example, a mapping of `kibana_user` (role) to `jdoe` (user) means that John Doe gains all the permissions of `kibana_user` after authenticating. Likewise, a mapping of `all_access` (role) to `admin` (backend role) means that any user with the backend role of `admin` gains all the permissions of `all_access` after authenticating. You can map each role to multiple users and/or backend roles.
```
Comment (kolchfa-aws) [inline _security/access-control/index.md:27]: I prefer "many" here because "one-to-many" is common for a database relationship.

**Review by kolchfa-aws — APPROVED** [review]
LGTM

**Review by cwillum — APPROVED** [review]

File: _security/access-control/index.md (hunk 1 above)
Comment (hdhalter) [reply _security/access-control/index.md:27]: This is mapping roles to users, though, and indicates "more than one". "Many" to me, makes me wonder how many is many? @natebower - do you have a preference?

Comment (natebower) [reply _security/access-control/index.md:27]: It depends on whether "a large number" (many) or "more than one" (multiple) is closer to our intended meaning, but they are nearly synonymous in this context.

**Pull request merged**